core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/DetailedSample.java

```diff
@@ -118,8 +118,20 @@
    */
   Optional<DetailedSample> getEffectiveGroupIdSample();
 
+  /**
+   * Field for storing the date of sample creation. This is not the date that the sample was entered into MISO.
+   * This field is not automatically generated on sample creation and must be specified by a user.
+   * 
+   * @return Date creationDate
```

Comment:
Return type should not be included in Javadoc (`@return <description>`). Same with parameter type in the setter (`@param <name> <description>`)